introduction.md

```diff
@@ -2,4 +2,11 @@ Introducing my self
 ===================
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?